cmd/draconctl/main.go

```diff
@@ -58,6 +66,12 @@ func initializeConfig(cmd *cobra.Command) error {
 	// like --favorite-color which we fix in the bindFlags function
 	v.AutomaticEnv()
 
+	if !slices.Contains(supportedLogFormats, strings.ToLower(loggingFormat)) {
+		return errors.Errorf("unsupported log format %s, supported formats are %s", loggingFormat, supportedLogFormats)
+	} else if strings.ToLower(loggingFormat) == "json" {
+		slog.SetDefault(slog.New(slog.NewJSONHandler(os.Stdout, &slog.HandlerOptions{})))
```

Comment:
since we are switching to `slog` we should also replace the `log` invocation a few lines below this